pyaedt/icepak.py

```diff
@@ -3968,18 +3918,13 @@ def assign_source(
         thermal_condition : str
             Thermal condition. Accepted values are ``"Total Power"``, ``"Surface Heat"``,
             ``"Temperature"``.
-        assignment_value : str or dict
+        assignment_value : str or dict or BoundaryDictionary
             Value and units of the input power, surface heat or temperature (depending on
```

Comment:
```suggestion
            Value and units of the input power, surface heat, or temperature (depending on
```
We use an Oxford comma, which says to always use a comma before the "and" or the "or" in a list of items.